README.md

```diff
@@ -11,7 +11,8 @@ There's also an
 Chrome and probably Safari).
 
 #### Note:
-- `uglify-js` only supports ECMAScript 5 (ES5). 
+- `uglify-js` only supports ECMAScript 5 (ES5).
+- Support for `const` is present but incomplete, and may not be transformed properly.
```

Comment:
No, just add that sentence after the one you have.

You could add an additional line:

It is recommended to avoid using `const` with `uglify-2.x` for this reason as it will have undefined behavior when run on an ES2015 compatible browser.